example/1_getstarted.html

```diff
@@ -1,114 +1,114 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<meta charset="utf-8">
-<title>=^.^=</title>
-<style>
-body{margin: 0;padding: 0; background: #000;}
-canvas{display: block;}
-.info{
-  color: #fff;
-  position: absolute;
-  top: 0;
-  left: 0;
-}
-</style>
+  <meta charset="utf-8">
+  <title>1: Getting Started</title>
+  <link rel="stylesheet" href="./css/examples.css">

```

Comment:
I was wondering if you could focus on main subject and purpose which support latest three.js, in this PR.

Otherwise, I cannot approve to merge until all change will be appropriated. I don't want to separate CSS from HTML source, as an example source, as well as some indent and spacing rules.🙇

Reply:
Heya I've been busy I was wondering if you've had a chance to look at the last commit?

Reply:
sorry for very late...
seems your commits still contains unnecessary changes 😢
Could you check [the diff](https://github.com/yomotsu/meshwalk.js/pull/3/files) please?